src/detect.h

```diff
@@ -716,6 +711,8 @@ typedef struct DetectEngineIPOnlyCtx_ {
     /* Used to build the radix trees */
     IPOnlyCIDRItem *ip_src, *ip_dst;
     uint32_t max_idx;
+    uint32_t *sig_mapping;
```

Comment:
can you add a description of what the mapping is?

Reply:
No problem.. also, that should just be defined as a SigIntId right? I realized that and the alloc call should probably be SigIntId and not use uint32 directly.

Reply:
Added comment

```
+    /* Used to map large signums to smaller values to compact the bitsets
+     * stored in the radix trees */
```

Still a little fuzzy on if i should change the uint32_t to be SigIntId or not.